one-ui/src/main/java/com/marklogic/hub/curation/controllers/LoadDataController.java

```diff
@@ -108,6 +108,19 @@ public String getArtifactType() {
         return "loadData";
     }
 
+    @RequestMapping(value = "/{artifactName}/settings", method = RequestMethod.GET)
+    @ResponseBody
+    public ResponseEntity<JsonNode> getArtifactSettings(@PathVariable String artifactName) {
+        ResponseEntity<JsonNode> resp = super.getArtifactSettings(artifactName);
```

Comment:
Tiny nit - I think it's better just to return the result, there's no value in defining a variable here and then returning it.

Reply:
Yeah, i forgot simplifying after testing.